src/convertToObject.js

```diff
@@ -1,12 +1,21 @@
-'use strict';
-
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styleString) {
+  const styleObject = {};
+  const lines = styleString.split(';');
+
+  lines.forEach((line) => {
+    const [property, value] = line.split(':');
```

Comment:
The split operation here assumes there will only be one colon ':' in each line. While this may work for most cases, it could lead to incorrect results if a value contains a colon. A more robust approach would be to use a regular expression or to find the index of the first colon and use 'substring' to separate the property and value.